src/ApiConnectors/BaseApiConnector.php

```diff
@@ -161,4 +166,70 @@ protected function getFinderService(): FinderService
     {
         return $this->connection->getAuthenticatedClient(Services::FINDER());
     }
-}
+
+    /**
+     * Convert options array to an ArrayOfString which is accepted by Twinfield.
+     *
+     * @param array $options
+     * @param array|null $forcedOptions
+     * @return array
+     * @throws Exception
+     */
+    public function convertOptionsToArrayOfString(array $options, array $forcedOptions = null): array {
```

Comment:
`$forcedOptions` can be defaulted to an `[]`, that allows you to skip the isset() check too.

Reply:
Done